Food_Tracker_App/app/src/main/res/layout/ingredient_content.xml

```diff
@@ -1,22 +1,107 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.cardview.widget.CardView
```

Comment:
I think since this is its own page generally it should not be elevated in a card view

Reply:
Can you also use OPTION + CMD + L on mac or CTRL + ALT + L on windows to reformat this file?